tests/common/docker_image.py

```diff
@@ -302,3 +305,88 @@ def wait(self, max_attempts: int = 15, sleep_rate: float = 1.0) -> bool:
                 )
                 time.sleep(sleep_rate)
         return False
+
+
+class FetchLedgerDockerImage(DockerImage):
+    """Wrapper to Fetch ledger Docker image."""
+
+    def __init__(
+        self,
+        client: DockerClient,
+        addr: str,
+        port: int,
+        config: Optional[Dict] = None,
+        gas_limit: int = 10000000000000,
+    ):
+        """
+        Initialize the Fetch ledger Docker image.
+
+        :param client: the Docker client.
+        :param addr: the address.
+        :param port: the port.
+        :param config: optional configuration to command line.
+        """
+        super().__init__(client)
+        self._addr = addr
+        self._port = port
+        self._config = config or {}
+        self._gas_limit = gas_limit
+
+    @property
+    def tag(self) -> str:
+        """Get the image tag."""
+        return "fetchai/fetchd:0.2.7"
+
+    def _make_ports(self) -> Dict:
+        """Make ports dictionary for Docker."""
+        return {f"{self._port}/tcp": ("0.0.0.0", self._port)}  # nosec
+
+    def _make_entrypoint_file(self, tmpdirname) -> None:
+        """Make a temporary entrypoint file to setup and run the test ledger node"""
+        run_node_lines = [
+            '#!/usr/bin/env bash',
+            'set -e',
+            'fetchd init test-node --chain-id test',
+            'sed -i "s/stake/atestfet/" ~/.fetchd/config/genesis.json',
+            'sed -i "s/enable = false/enable = true/" ~/.fetchd/config/app.toml',
+            'MNEMONIC="gap bomb bulk border original scare assault pelican resemble found laptop skin gesture height inflict clinic reject giggle hurdle bubble soldier hurt moon hint"',
+            'fetchcli config keyring-backend test',
+            'echo $MNEMONIC | fetchcli keys add validator --recover',
+            'fetchd add-genesis-account $(fetchcli keys show validator -a) 1152997575000000000000000000atestfet',
```

Comment:
Need to sync that with the keys we have in conftest...

Reply:
Do we have the mnemonics for the existing keys somewhere?